---
tags:
  - _durability/fleeting
aliases:
  - tools
cssclasses:
  - idea
publish: true
---
# Bioinformatical Tools

## Trimmomatic: a flexible trimmer for Illumina sequence data
**Authors:** A. M. Bolger; M. Lohse; B. Usadel
**Journal:** Bioinformatics
**Year:** 2014
**DOI:** [10.1093/bioinformatics/btu170](https://doi.org/10.1093/bioinformatics/btu170)
**Languages:** [[Java]]
**Source:** [USADEL Lab](http://www.usadellab.org/cms/index.php?page=trimmomatic)
**Tutorial:** [USADEL Lab](http://www.usadellab.org/cms/index.php?page=trimmomatic)

## STAR: ultrafast universal RNA-seq aligner
**Authors:** A. Dobin; C. A. Davis; F. Schlesinger; J. Drenkow; C. Zaleski; S. Jha; P. Batut; M. Chaisson; T. R. Gingeras
**Journal:** Molecular Ecology Resources
**Year:** 2013
**DOI:** [10.1093/bioinformatics/bts635](https://doi.org/10.1093/bioinformatics/bts635)
**Languages:** [[C++]]
**Source:** [GitHub](https://github.com/alexdobin/STAR)
**Tutorial:** [GitHub.com](https://github.com/alexdobin/STAR/blob/master/doc/STARmanual.pdf)

## vcfr: a package to manipulate and visualize variant call format data in R
**Authors:** B. J. Knaus; N. J. Grünwald
**Journal:** Molecular Ecology Resources
**Year:** 2016
**DOI:** [10.1111/1755-0998.12549](https://doi.org/10.1111/1755-0998.12549)
**Languages:** [[R]]
**Source:** [GitHub](https://github.com/knausb/vcfR)
**Tutorial:** [GitHub.io](https://cran.r-project.org/web/packages/vcfR/vignettes/intro_to_vcfR.html)

## Trim Galore
A wrapper tool around Cutadapt and FastQC to consistently apply quality and adapter trimming to FastQ files, with some extra functionality for MspI-digested RRBS-type (Reduced Representation Bisufite-Seq) libraries.

**Authors:** Krueger, F 
**Journal:** N/A
**Year:** 2015
**DOI:** N/A
**Languages:** [[Perl]]
**Source:** [GitHub](https://github.com/FelixKrueger/TrimGalore)
**Tutorial:** N/A
**Uses:** [Cutadapt](M-Martin-2010.md), [FastQC](S-Andrews-2010.md)

## Cutadapt Removes Adapter Sequences From High-Throughput Sequencing Reads
**Authors:** M. Martin
**Journal:** EMBnet Journal
**Year:** 2011
**DOI:** [10.14806/ej.17.1.200](https://doi.org/10.14806/ej.17.1.200)
**Languages:** [[Python]]
**Source:** [GitHub](https://github.com/marcelm/cutadapt)
**Tutorial:** [ReadTheDocs.io](https://cutadapt.readthedocs.io/)

## miRDeep2 accurately identifies known and hundreds of novel microRNA genes in seven animal clades
**Authors:** M. R. Friedländer; S. D. Mackowiak; N. Li; W. Chen; N. Rajewsky
**Journal:** Nucleic Acids Research
**Year:** 2012
**DOI:** [10.1093/nar/gkr688](https://doi.org/10.1093/nar/gkr688)
**Languages:** [[Perl]]
**Source:** [GitHub](https://github.com/rajewsky-lab/mirdeep2)
**Patch:** [GitHub](https://github.com/Drmirdeep/mirdeep2_patch)
**Tutorial:** [GitHub.io](https://drmirdeep.github.io/mirdeep2_tutorial.html)

## A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w<sup>1118</sup>; iso-2; iso-3
**Authors:** P. Cingolani; A. Platts; L. Lily Wang; M. Coon; T. Nguyen; Luan Wang; S. J. Land; D. M. Ruden; X. Lu
**Journal:** Landes Bioscience
**Year:** 2016
**DOI:** [10.4161/fly.19695](http://dx.doi.org/10.4161/fly.19695)
**Languages:** [[Java]]
**Source:** [GitHub](https://github.com/pcingola/SnpEff)
**Tutorial:** [GitHub.io](https://pcingola.github.io/SnpEff/se_introduction/)

## HTSeq - a Python framework to work with high-throughput sequencing data
**Authors:** S. Anders; P. T. Pyl; W. Huber
**Journal:** Bioinformatics
**Year:** 2015
**DOI:** [10.1093/bioinformatics/btu638](https://doi.org/10.1093/bioinformatics/btu638)
**Languages:** [[Python]]
**Source:** [GitHub](https://github.com/htseq/htseq)
**Tutorial:** [ReadTheDocs.io](http://htseq.readthedocs.io)

## vcfr: a package to manipulate and visualize variant call format data in R
**Authors:** S. Andrews
**Journal:** N/A
**Year:** 2010
**DOI:** N/A
**Languages:** [[Java]]
**Download:** [Babraham](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/)
**Source:** N/A
**Tutorial:** N/A

## miRBase: microRNA sequences, targets and gene nomenclature
**Authors:** S. Griffiths-Jones; R. J. Grocock; S. van Dongen; A. Bateman; A. J. Enright
**Journal:** Nucleic Acids Research
**Year:** 2006
**DOI:** [10.1093/nar/gkj112](https://doi.org/10.1093/nar/gkj112)
**Languages:** N/A
**Source:** [Website](https://www.mirbase.org)
**Tutorial:** N/A
**Reference data**: [[reference-data#^0873a0]]
