---
tags: üèõ üî¨ ‚úçÔ∏è
aliases:
  - bibikova2009
  - bibikovaGenomewideDNAMethylation2009
cssclass: article
publish: true
authors:
  - Marina Bibikova
  - Jennie Le
  - Bret Barnes
  - Shadi Saedinia-Melnyk
  - Lixin Zhou
  - Richard Shen
  - Kevin L Gunderson
article-type: research
journal: Epigenomics
year: 2009
cohorts: none
---
# Genome-wide DNA methylation profiling using Infinium <sup>¬Æ</sup> assay
**DOI:** [10.2217/epi.09.14](https://www.doi.org/10.2217/epi.09.14)
**Zotero:** [Click here](zotero://select/items/@bibikovaGenomewideDNAMethylation2009)

## Abstract
Over here I should write shortly what this paper is about, their hypothesis and their main findings
> Aims: Bisulfite sequence analysis of individual CpG sites within genomic DNA is a powerful approach for methylation analysis in the genome. The major limitation of bisulfite-based methods is parallelization. Both array and next-generation sequencing technology are capable of addressing this bottleneck. In this report, we describe the application of Infinium ¬Æ genotyping technology to analyze bisulfite-converted DNA to simultaneously query the methylation state of over 27,000 CpG sites from promoters of consensus coding sequences (CCDS) genes. Materials & methods: We adapted the Infinium genotyping assay to readout an array of over 27,000 pairs of CpG methylation-specific query probes complementary to bisulfite-converted DNA. Two probes were designed to each CpG site: a ‚Äòmethylated‚Äô and an ‚Äòunmethylated‚Äô query probe. The probe design assumed that all underlying CpG sites were ‚Äòin phase‚Äô with the queried CpG site due to their close proximity. Bisulfite conversion was performed with a modified version of the Zymo EZ DNA Methylation‚Ñ¢ kit. Results: We applied this technology to measuring methylation levels across a panel of 14 different human tissues, four Coriell cell lines and six cancer cell lines. We observed that CpG sites within CpG islands (CGIs) were largely unmethylated across all tissues (‚àº80% sites unmethylated, Œ≤ < 0.2), whereas CpG sites in non-CGIs were moderately to highly methylated (only ‚àº12% sites unmethylated, Œ≤ < 0.2). Within CGIs, only approximately 3‚Äì6% of the loci were highly methylated; in contrast, outside of CGIs approximately 25‚Äì40% of loci were highly methylated. Moreover, tissue-specific methylation (variation in methylation across tissues) was much more prevalent in non-CGIs than within CGIs. Conclusion: Our results demonstrate a genome-wide scalable array-based methylation readout platform that is both highly reproducible and quantitative. In the near future, this platform should enable the analysis of hundreds of thousands to millions of CpG sites per sample.

## Methods
...

## Results
...

## Conclusion
...

## Discussion
...

## References
 - [[@pidsleyDatadrivenApproachPreprocessing2013]]