---
tags: _type/tool
title: "Bump hunting to identify differentially methylated regions in epigenetic epidemiology studies"
aliases:
  - jaffe2012
  - jaffeBumpHuntingIdentify2012
cssclass: article
publish: true
authors:
  - Andrew E Jaffe
  - Peter Murakami
  - Hwajin Lee
  - Jeffrey T Leek
  - M Daniele Fallin
  - Andrew P Feinberg
  - Rafael A Irizarry
journal: International Journal of Epidemiology
year: 2012
cohorts: none
---
# Bump hunting to identify differentially methylated regions in epigenetic epidemiology studies
**DOI:** [10.1093/ije/dyr238](https://www.doi.org/10.1093/ije/dyr238)
**Zotero:** [Click here](zotero://select/items/@jaffeBumpHuntingIdentify2012)
**Authors:**
  - Andrew E Jaffe
  - Peter Murakami
  - Hwajin Lee
  - Jeffrey T Leek
  - M Daniele Fallin
  - Andrew P Feinberg
  - Rafael A Irizarry

## Abstract
Over here I should write shortly what this paper is about, their hypothesis and their main findings
> Background During the past 5 years, high-throughput technologies have been successfully used by epidemiology studies, but almost all have focused on sequence variation through genome-wide association studies (GWAS). Today, the study of other genomic events is becoming more common in large-scale epidemiological studies. Many of these, unlike the single-nucleotide polymorphism studied in GWAS, are continuous measures. In this context, the exercise of searching for regions of interest for disease is akin to the problems described in the statistical ‘bump hunting’ literature. Methods New statistical challenges arise when the measurements are continuous rather than categorical, when they are measured with uncertainty, and when both biological signal, and measurement errors are characterized by spatial correlation along the genome. Perhaps the most challenging complication is that continuous genomic data from large studies are measured throughout long periods, making them susceptible to ‘batch effects’. An example that combines all three characteristics is genome-wide DNA methylation measurements. Here, we present a data analysis pipeline that effectively models measurement error, removes batch effects, detects regions of interest and attaches statistical uncertainty to identified regions. Results We illustrate the usefulness of our approach by detecting genomic regions of DNA methylation associated with a continuous trait in a well-characterized population of newborns. Additionally, we show that addressing unexplained heterogeneity like batch effects reduces the number of false-positive regions. Conclusions Our framework offers a comprehensive yet flexible approach for identifying genomic regions of biological interest in large epidemiological studies using quantitative high-throughput methods.

## Methods
...

## Results
...

## Conclusion
...

## Discussion
...

## References
...