---
tags: _type/article _type/tool
title: "Comparison of novel and existing methods for detecting differentially methylated regions"
aliases:
  - lent2018
  - lentComparisonNovelExisting2018
cssclass: article
publish: true
authors:
  - Samantha Lent
  - Hanfei Xu
  - Lan Wang
  - Zhe Wang
  - Chloé Sarnowski
  - Marie-France Hivert
  - Josée Dupuis
journal: BMC Genetics
year: 2018
cohorts: none
---
# Comparison of novel and existing methods for detecting differentially methylated regions
**DOI:** [10.1186/s12863-018-0637-4](https://www.doi.org/10.1186/s12863-018-0637-4)
**Zotero:** [Click here](zotero://select/items/@lentComparisonNovelExisting2018)
**Authors:**
  - Samantha Lent
  - Hanfei Xu
  - Lan Wang
  - Zhe Wang
  - Chloé Sarnowski
  - Marie-France Hivert
  - Josée Dupuis

## Abstract
Over here I should write shortly what this paper is about, their hypothesis and their main findings
> Background: Single-probe analyses in epigenome-wide association studies (EWAS) have identified associations between DNA methylation and many phenotypes, but do not take into account information from neighboring probes. Methods to detect differentially methylated regions (DMRs) (clusters of neighboring probes associated with a phenotype) may provide more power to detect associations between DNA methylation and diseases or phenotypes of interest. Results: We proposed a novel approach, GlobalP, and perform comparisons with 3 methods—DMRcate, Bumphunter, and comb-p—to identify DMRs associated with log triglycerides (TGs) in real GAW20 data before and after fenofibrate treatment. We applied these methods to the summary statistics from an EWAS performed on the methylation data. Comb-p, DMRcate, and GlobalP detected very similar DMRs near the gene CPT1A on chromosome 11 in both the preand posttreatment data. In addition, GlobalP detected 2 DMRs before fenofibrate treatment in the genes ETV6 and ABCG1. Bumphunter identified several DMRs on chromosomes 1 and 20, which did not overlap with DMRs detected by other methods. Conclusions: Our novel method detected the same DMR identified by two existing methods and detected two additional DMRs not identified by any of the existing methods we compared.

## Methods
...

## Results
...

## Conclusion
...

## Discussion
...

## References
...