---
tags: _type/tool
title: "De novo identification of differentially methylated regions in the human genome"
aliases:
  - peters2015
  - petersNovoIdentificationDifferentially2015
cssclass: article
publish: true
authors:
  - Timothy J Peters
  - Michael J Buckley
  - Aaron L Statham
  - Ruth Pidsley
  - Katherine Samaras
  - Reginald V Lord
  - Susan J Clark
  - Peter L Molloy
journal: Epigenetics & Chromatin
year: 2015
cohorts: none
---
# De novo identification of differentially methylated regions in the human genome
**DOI:** [10.1186/1756-8935-8-6](https://www.doi.org/10.1186/1756-8935-8-6)
**Zotero:** [Click here](zotero://select/items/@petersNovoIdentificationDifferentially2015)
**Authors:**
  - Timothy J Peters
  - Michael J Buckley
  - Aaron L Statham
  - Ruth Pidsley
  - Katherine Samaras
  - Reginald V Lord
  - Susan J Clark
  - Peter L Molloy

## Abstract
Over here I should write shortly what this paper is about, their hypothesis and their main findings
> Abstract Background: The identification and characterisation of differentially methylated regions (DMRs) between phenotypes in the human genome is of prime interest in epigenetics. We present a novel method, DMRcate,thatfits replicated methylation measurements from the Illumina HM450K BeadChip (or 450K array) spatially across the genome using a Gaussian kernel. DMRcate identifies and ranks the most differentially methylated regions across the genome based on tunable kernel smoothing of the differential methylation (DM) signal. The method is agnostic to both genomic annotation and local change in the direction of the DM signal, removes the bias incurred from irregularly spaced methylation sites, and assigns significance to each DMR called via comparison to a null model. Results: We show that, for both simulated and real data, the predictive performance of DMRcate is superior to those of Bumphunter and Probe Lasso, and commensurate with that of comb-p. For the real data, we validate all array-derived DMRs from the candidate methods on a suite of DMRs derived from whole-genome bisulfite sequencing called from the same DNA samples, using two separate phenotype comparisons. Conclusions: The agglomeration of genomically localised individual methylation sites into discrete DMRs is currently best served by a combination of DM-signal smoothing and subsequent threshold specification. The findings also suggest the design of the 450K array shows preference for CpG sites that are more likely to be differentially methylated, but its overall coverage does not adequately reflect the depth and complexity of methylation signatures afforded by sequencing. For the convenience of the research community we have created a user-friendly R software package called DMRcate, downloadable from Bioconductor and compatible with existing preprocessing packages, which allows others to apply the same DMR-finding method on 450K array data.

## Methods
...

## Results
...

## Conclusion
...

## Discussion
...

## References
...