---
tags: ðŸš€
aliases:
  - rauluseviciute2019
  - rauluseviciuteDNAMethylationData2019
cssclass: article
publish: true
authors:
  - Ieva Rauluseviciute
  - Finn DrablÃ¸s
  - Morten Beck Rye
journal: Clinical Epigenetics
year: 2019
cohorts: none
---
# DNA methylation data by sequencing: Experimental approaches and recommendations for tools and pipelines for data analysis
**DOI:** [10.1186/s13148-019-0795-x](https://www.doi.org/10.1186/s13148-019-0795-x)
**Zotero:** [Click here](zotero://select/items/@rauluseviciuteDNAMethylationData2019)

## Abstract
Over here I should write shortly what this paper is about, their hypothesis and their main findings
> Abstract Sequencing technologies have changed not only our approaches to classical genetics, but also the field of epigenetics. Specific methods allow scientists to identify novel genome-wide epigenetic patterns of DNA methylation down to single-nucleotide resolution. DNA methylation is the most researched epigenetic mark involved in various processes in the human cell, including gene regulation and development of diseases, such as cancer. Increasing numbers of DNA methylation sequencing datasets from human genome are produced using various platformsâ€”from methylated DNA precipitation to the whole genome bisulfite sequencing. Many of those datasets are fully accessible for repeated analyses. Sequencing experiments have become routine in laboratories around the world, while analysis of outcoming data is still a challenge among the majority of scientists, since in many cases it requires advanced computational skills. Even though various tools are being created and published, guidelines for their selection are often not clear, especially to non-bioinformaticians with limited experience in computational analyses. Separate tools are often used for individual steps in the analysis, and these can be challenging to manage and integrate. However, in some instances, tools are combined into pipelines that are capable to complete all the essential steps to achieve the result. In the case of DNA methylation sequencing analysis, the goal of such pipeline is to map sequencing reads, calculate methylation levels, and distinguish differentially methylated positions and/or regions. The objective of this review is to describe basic principles and steps in the analysis of DNA methylation sequencing data that in particular have been used for mammalian genomes, and more importantly to present and discuss the most pronounced computational pipelines that can be used to analyze such data. We aim to provide a good starting point for scientists with limited experience in computational analyses of DNA methylation and hydroxymethylation data, and recommend a few tools that are powerful, but still easy enough to use for their own data analysis.

## Methods
...

## Results
...

## Conclusion
...

## Discussion
...

## References
...