---
tags: ðŸ› ðŸ”¬ âœï¸ ðŸ”– 
aliases:
  - suderman2018
  - sudermanDmrffIdentifyingDifferentially2018
cssclass: article
publish: true
authors:
  - Matthew Suderman
  - James R Staley
  - Robert French
  - Ryan Arathimos
  - Andrew Simpkin
  - Kate Tilling
article-type: research
journal: 
year: 2018
cohorts: none
---
# Dmrff: Identifying differentially methylated regions efficiently with power and control
**DOI:** [10.1101/508556](https://www.doi.org/10.1101/508556)
**Zotero:** [Click here](zotero://select/items/@sudermanDmrffIdentifyingDifferentially2018)

## Abstract
Over here I should write shortly what this paper is about, their hypothesis and their main findings
> A differentially methylated region (DMR) is a genomic region in which DNA methylation is consistently positively or negatively associated with a phenotype or exposure. We demonstrate that existing algorithms for identifying DMRs either fail to consistently control false positive rates (comb-p and DMRcate), suffer from low power (bumphunter) or lack modeling flexibility (seqlm). We introduce a new method, dmrff, that overcomes these shortcomings and can additionally be used to meta-analyze multiple datasets. When applied to investigate associations of age in multiple datasets, dmrff identifies novel DMRs near genes previously linked to age. An R implementation is available on Github (http://github.com/perishky/dmrff).

## Methods
...

## Results
...

## Conclusion
...

## Discussion
...

## References
...